### ISSUE TITLE
Trying to copy an image across to another subscription

### ISSUE THREAD

**Issue by noxx5489** [Azure/azure-cli-extensions#997]
## Describe the bug

**Command Name**
`az image copy
Extension Name: image-copy-extension. Version: 0.2.1.`

**Errors:**
```
Command '['/opt/az/bin/python3', '-m', 'azure.cli', 'snapshot', 'create', '--name', 'NC24rs-image-2019100209_os_disk_snapshot', '--resource-group', 'CycleCloudResourceGroup', '--source', '/subscriptions/d57b176e-d3ad-46c0-8f9d-33ab0cd2af37/resourceGroups/NC24RS-DEV3/providers/Microsoft.Compute/disks/NC24rs-dev3_OsDisk_1_0fdbbdd1b540410dbb69f03567302970', '--output', 'json', '--tags', 'created_by=image-copy-extension']' returned non-zero exit status 1.
Traceback (most recent call last):
...
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az image copy --source-resource-group {} --source-object-name {} --source-type {} --target-location {} --target-subscription {} --target-resource-group {}`

## Environment Summary
```
Linux-4.15.0-1060-azure-x86_64-with-debian-stretch-sid
Python 3.6.5
Shell: bash

azure-cli 2.0.74

Extensions:
image-copy-extension 0.2.1

```

**Comment by tamirkamara** [comment]
@noxx5489 please provide more information like debug information from running the command with the --debug flag.

**Comment by rodrgeve** [comment]
@tamirkamara - Here is a snippet of the error I get when running the command (--debug enabled):
```

command ended with an error: ['/azure-cli/bin/python', '-m', 'azure.cli', 'storage', 'account', 'create', '--name', 'canadacentral680bqpiaoa4', '--resource-group', 'image-copy-rg', '--location', 'canadacentral', '--sku', 'Standard_LRS', '--output', 'json', '--tags', 'created_by=image-copy-extension']

Event: CommandInvoker.OnTransformResult [<function _resource_group_transform at 0x7f1195703de8>, <function _x509_from_base64_to_hex_transform at 0x7f1195703e60>]

Event: CommandInvoker.OnFilterResult []

Event: Cli.PostExecute []

az_command_data_logger : exit code: 0
Suppress exception Return value is not string

telemetry.save : Save telemetry record of length 2503 in cache

telemetry.check : Negative: The /root/.azure/telemetry.txt was modified at 2019-10-16 13:59:21.760787, which in less than 600.000000 s
command ran in 61.454 seconds.

```

**Comment by tamirkamara** [comment]
Hi,
Looks like a duplicate of https://github.com/Azure/azure-cli-extensions/issues/987.
Can you check the issue+solution there and close this one if the same? Or report otherwise...

**Comment by tamirkamara** [comment]
@rodrgeve I believe this issue is resolved with version 0.2.2 of the extension.
@zikalino please close it.
